3_extract/src/process_glm_output.R

```diff
@@ -25,18 +25,16 @@ combine_glm_output <- function(run_group, outfile_template) {
   # saving only the temperature predictions and ice flags, and
   # truncating output for each time period to valid dates
   # (excluding burn-in and burn-out periods)
-  purrr::map2_df(run_group$raw_meteo_fl, run_group$export_fl, function(raw_meteo_fl, export_file) {
-    # Define time period begin and end dates from raw meteo_fl
-    meteo_data <- arrow::read_feather(raw_meteo_fl, col_select = "time")
-    begin <- min(meteo_data$time)
-    end <- max(meteo_data$time)
+  purrr::pmap_df(run_group, function(...) {
```

Comment:
this sure beats writing out all the columns. I will be using this the next time I need to do something similar